todo/src/reducers/todos.js

```diff
@@ -0,0 +1,37 @@
+import { ADD_TODO, TOGGLE_TODO } from '../actions';
+
+const initialState = {
+    todos: [
+        { todo: 'Learn React', completed: false },
+        { todo: 'Learn Redux', completed: false },
+        { todo: 'Watch Anime', completed: false }
+    ]
+}
+
+export const todos = (state = initialState, action) => {
+    switch(action.type) {
+        case ADD_TODO:
+            const newTodo = { name: action.payload, completed: false };
```

Comment:
const newTodo = { todo: action.payload, completed: false };

[Click me!](http://prntscr.com/o3ms4u)

Reply:
fixed it. Thanks!